nixos/modules/installer/tools/tools.nix

```diff
@@ -126,7 +126,7 @@ in
       # your system.  Help is available in the configuration.nix(5) man page
       # and in the NixOS manual (accessible by running `nixos-help`).
 
-      { config, pkgs, ... }:
+      { config, lib, pkgs, ... }:
```

Comment:
Why not go all the way and put `options, utils` and add a comment to document them?

Reply:
`options` is basically only useful to module authors and probably distracting for beginners, and I think `utils` is [frowned upon](https://github.com/NixOS/nixpkgs/pull/160489#issuecomment-1049692147)

Reply:
Got it